arches/app/templates/views/resource/editor.htm

```diff
@@ -181,6 +181,7 @@ <h1 class="page-header text-overflow ep-graph-title">
                     <div class="card-breadcrumbs">
 
                         <a href="#" class="toggle-tree"><i class="ion-arrow-left-b"></i></a>
+                        <!--<span class="toggle-tree"><i class="ion-arrow-left-b"></i></span>-->
```

Comment:
Is this commented out what should be replacing the <a> ??

Reply:
Yes, must have missed this, well spotted!!! I've sorted this in the latest commit